webpack.dev.config.js

```diff
@@ -10,11 +10,52 @@ BaseConfig.entry = [
 BaseConfig.output = {
   path: join(__dirname, 'build'),
   filename: 'bundle.js',
+  publicPath: '/',  
+};
+
+BaseConfig.devServer = {
+  historyApiFallback: true,
 };
 
 BaseConfig.plugins.push(new HtmlWebpackPlugin({
   filename: 'index.html',
   template: './example/src/index.html',
 }));
+/*
+BaseConfig.module.rules.splice(2, 1, {
```

Comment:
@mkulinski Yes. Now, we have proper resolution of that error, I removed comments.